docs/snippets/tracer/captureMethodDecorator.ts

```diff
@@ -4,16 +4,16 @@ import { LambdaInterface } from '@aws-lambda-powertools/commons';
 const tracer = new Tracer({ serviceName: 'serverlessAirline' });
 
 class Lambda implements LambdaInterface {
-    // Decorate your class method
-    @tracer.captureMethod() // (1)
-    public async getChargeId(): Promise<string> {
-        /* ... */
-        return 'foo bar';
-    }
+  // Decorate your class method
+  @tracer.captureMethod() // (1)
+  public async getChargeId(): Promise<string> {
+    /* ... */
+    return 'foo bar';
+  }
 
-    public async handler(_event: any, _context: any): Promise<void> {
-        /* ... */
-    }
+  public async handler(_event: any, _context: any): Promise<void> {
+    /* ... */
```

Comment:
Let's remove this empty comment line and call the other method from the class instead `await this.getChargeId()`